checkClock/app/controllers/home_controller.rb

```diff
@@ -61,4 +61,11 @@ def out_employee
     end
   end
 
+  def getBranches 
+    branches = Branch.all
```

Comment:
Why making this, instead of loading branches in the `index` view

Just curious?